mu4e/mu4e-mark.el

```diff
@@ -266,8 +266,8 @@ The following marks are available, and the corresponding props:
         ;; update the hash -- remove everything current, and if add the new
         ;; stuff, unless we're unmarking
         (remhash docid mu4e~mark-map)
-        ;; remove possible overlays
-        (remove-overlays (line-beginning-position) (line-end-position))
+        ;; remove possible mark overlays
+        (remove-overlays (line-beginning-position) (line-end-position) 'mmark t)
         ;; now, let's set a mark (unless we were unmarking)
```

Comment:
What does `mmark` mean? If it's short for `mu4e-mark`, can you rename it to that?